backend/routes/api/quickstarts/quickStartUtils.js

```diff
@@ -19,19 +22,21 @@ const getInstalledQuickStarts = async (fastify) => {
     installedQuickStarts.push(...res.body.items);
   } catch (e) {
     fastify.log.error(e, 'failed to get quickstarts');
-    const error = createError(500, 'failed to get quickstarts');
-    error.explicitInternalServerError = true;
-    error.error = 'failed to get quickstarts';
-    error.message =
-      'Unable to load Kubeflow resources. Please ensure the Open Data Hub operator has been installed.';
-    throw error;
+    if (!constants.USE_MOCK_DATA || USE_LOCAL_QUICK_STARTS) {
+      const error = createError(500, 'failed to get quickstarts');
+      error.explicitInternalServerError = true;
+      error.error = 'failed to get quickstarts';
+      error.message =
+        'Unable to load Kubeflow resources. Please ensure the Open Data Hub operator has been installed.';
+      throw error;
+    }
   }
 
-  // TODO: Remove
-  if (constants.DEV_MODE) {
+  if (USE_LOCAL_QUICK_STARTS || constants.USE_MOCK_DATA) {
```

Comment:
review: `USE_LOCAL_QUICK_STARTS` is always true?

Reply:
Yes, eventually it will be removed once we get quick starts from openshift.

Reply:
I've removed the `USE_LOCAL_QUICK_STARTS` constant and changed code to work as if it is always true (which it was 😉 )